plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -37,7 +37,7 @@ export class Connect extends Component {
 	}
 
 	async connectJetpack() {
-		const { jetpackConnectUrl, onConnect } = this.props;
+		const { jetpackAuthUrl, onConnect } = this.props;
```

Comment:
The onboarding data store calls this `jetpackAuthUrl` so I stuck with this naming here.